tests/Extensions/Hosting.Services.UnitTests/ServiceAccessorTests.cs

```diff
@@ -7,6 +7,8 @@
 
 namespace Hosting.Services.UnitTests
 {
+
+
 	[TestClass]
 	public class ServiceAccessorTests
```

Comment:
The changes to this file appear to be by mistake

Reply:
Yes, removed, thank you